packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -160,13 +159,21 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
       timestamps[token].getHead(),
       address(0)
     );
-    emit OracleReported(token, msg.sender, now, value, DENOMINATOR);
+    emit OracleReported(token, msg.sender, now, fractionalValueReported.unwrap(), getDenominator());
     uint256 newMedian = rates[token].getMedianValue();
     if (newMedian != originalMedian) {
-      emit MedianUpdated(token, newMedian, DENOMINATOR);
+      emit MedianUpdated(token, newMedian, getDenominator());
     }
   }
 
+  /**
+   * @notice Returns the common fixidity denominator.
+   * @return The common fixidity denominator.
+   */
+  function getDenominator() public view returns (uint256) {
```

Comment:
Do we need this?

Reply:
since we have the int. denominator as private, I have added a public method that is to be used by external sources e.g. contractskit and cli. Please let me know if any objections to that